DataFormats/simulation/include/SimulationDataFormat/Stack.h

```diff
@@ -281,6 +282,8 @@ class Stack : public FairGenericStack
 
   bool mIsExternalMode = false; // is stack an external factory or directly used inside simulation?
 
+  std::function<bool(const TParticle& p)> mInhibitPrimary = [](const TParticle& p) { return false; }; // a function to inhibit the tracking of a particle
```

Comment:
there is a missing include here

Reply:
uhm... yes you are right (why did it build on my computer?)